src/dataDisplay/personPanel/personPanelSummary.jsx

```diff
@@ -530,62 +532,63 @@ function PersonPanelSummary(props) {
             tabIndex={tabIndex}
             role="button"
         >
-            <Grid className={classes.grid}>
-                <Grid.Row
-                    className={classes.gridRow}
+            <Grid
+                alignItems="center"
+                className={classes.grid}
+                spacing={0}
+                wrap="nowrap"
+            >
+                <Grid.Column
+                    className={classes.avatarColumn}
                 >
-                    <Grid.Column
-                        className={classes.avatarColumn}
+                    <Image
+                        className={classes.avatar}
+                        name={firstName && lastName ? `${firstName} ${lastName}` : null}
+                        size={44}
+                        src={avatar}
+                        type="person"
+                    />
+                </Grid.Column>
+
+                <Grid.Column
+                    className={classes.nameColumn}
+                >
+                    <Typography
+                        component="h4"
```

Comment:
Is `component="h4"` needed here?  I imagine that's what you'd expect by default with `variant="h4"`, right?

Reply:
Fixed.

Reply:
Resolving.